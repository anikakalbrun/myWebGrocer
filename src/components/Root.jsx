import React from 'react';
import 'bootstrap/dist/css/bootstrap.min.css';

import Header from './Header';
import Main from '../containers/Main';

export default function Root() {
    return (
        <div>
            <Header/>
            <Main/>
        </div>
    );
}