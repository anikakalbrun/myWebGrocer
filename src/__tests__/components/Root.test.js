import React from 'react';

import Root from '../../components/Root';

let wrapper;

describe('Root', () => {

    beforeEach(() => {
        wrapper = shallow(<Root/>);
    });

    it('renders the Header component', () => {
        expect(wrapper.find('Header').length).toEqual(1)
    });

    it('renders the Main component', () => {
        expect(wrapper.props().children[1]).toMatchSnapshot()
    });
});
