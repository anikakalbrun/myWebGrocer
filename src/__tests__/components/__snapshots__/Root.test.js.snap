// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root renders the Main component 1`] = `<Connect(Main) />`;
